src/features/books/components/AddBookForm/AddBookForm.tsx

```diff
@@ -37,137 +47,149 @@ const AddBookForm = () => {
   useEffect(() => {
     if (!genres) {
       dispatch(getAllGenres())
-      dispatch(getAllBooks())
     }
   }, [dispatch, genres])
 
   if (isGenresLoading || !genres)
```

Comment:
I think it would be better to do this logic in the parent component. Since add book form component should only render a form. 

Also when doing that you can also create a prop genres which this form will receive